ci/pipelines/concourse-pypi-resource-pipeline-template.yml

```diff
@@ -161,7 +161,6 @@ jobs:
     inputs: [image, version]
```

Comment:
you can remove version here. No need to stream if its unused.

Reply:
Absolutely, it's not used